tests/CleaningTheToilet.tex

```diff
@@ -0,0 +1,89 @@
+\section{Filling the Dishwasher [DSPL \& OPL]}
+The robot is able to remove dirt from a toilet using a brush or a sponge.
+
+\subsection{Goal}
+The robot has to correctly identify a dark spot over a white surface close to water and rub it with an appropriate tool until is removed.
+
+\subsection{Focus}
+This test focuses on spot recognition (substances, texture and color) as well as manipulation in narrow spaces using specialized tools.
+
+\subsection{Setup}
+\begin{enumerate}
+
+	\item \textbf{Location:} This test takes place in the bathroom. 
+	\item \textbf{Start position:} The robot starts outside the bathroom next to the door.
+	\item \textbf{Toilet:} The toilet is partially full with water. Close to the water surface there is an artificial spot (chocolate syrup, peanut butter, jam, etc). There is no water in the deposit.
+	\item \textbf{Tool:} A standard toilet-cleaning brush or sponge is located next to the toilet.
+\end{enumerate}
+
+
+\subsection{Task}
+The robot must execute the following tasks in any logical order, aiming to remove the spot.
+\begin{itemize}
+	\item \textbf{Enter the bathroom:} The robot locates itself in the apartment and navigates inside the bathroom.
+	
+	\item \textbf{Identify the spot:} The robot inspects the toilet and finds the spot, pointing at it, and stating that requires to be cleaned.
+	
+	\item \textbf{Grasp the tool:} The robot locates and grasps the cleaning tool, lifting is from its base and placing the cleaner inside the toilet.
+	
+	\item \textbf{Clean the spot:} The robot makes contact with the spot and rubs it until it has disappeared.
+
+	\item \textbf{Flush:} After cleaning the spot, the robot flushes the toilet, rinsing the tool.
+
+	\item \textbf{Place the tool:} The robot returns the cleaning tool to its base.
+
+	\item \textbf{Leave the bathroom:} After completing the cleaning, the robot leaves bathroom.
+\end{itemize}
+
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}
+	\item \textbf{Bypassing Manipulation:} Bypassing object manipulation via the CONTINUE rule (Section \refsec{rule:mancontinue}) is not allowed during this test.
+	\item \textbf{No setup:} There is no setup time.
```

Comment:
For logistics, having the bathroom besides a corridor would be handy. The robots line up in the corridor in one direction, get out towards the other end.

Reply:
That should go in the Scenario section, but I don't want to constrain how the arena should look like (e.g. in many places around the world, the toilet room is outside the house in the yard).